azure-iot-hub-devicesdk/samples/x509_send_telemetry.py

```diff
@@ -0,0 +1,40 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import os
+import time
+from azure.iot.hub.devicesdk.device_client import DeviceClient
+from azure.iot.hub.devicesdk.auth.authentication_provider_factory import from_x509
+
+
+# X509 certificate and key paths, and optional passphrase
+cert_file = "./scripts/certificate.pem"
+key_file = "./scripts/key.pem"
+passphrase = "passphrase"
+
+# The "Authentication Provider" will hold references to the device identity and certificate information and provide them to the transport when necessary
+x509_auth_provider = from_x509(os.getenv("DEVICE_ID"), os.getenv("HOSTNAME"), cert_file, key_file, passphrase)
```

Comment:
>DEVICE_ID [](start = 42, length = 9)

Let's standardize on environment variable names.  We use IOTHUB_DEVICE_ and IOTHUB_MODULE_ as prefixes in the other samples and we should be consistent.